src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -33,8 +32,11 @@ internal class AzureAppConfigurationProvider : ConfigurationProvider, IConfigura
         private AzureAppConfigurationOptions _options;
         private Dictionary<string, ConfigurationSetting> _mappedData;
         private Dictionary<KeyValueIdentifier, ConfigurationSetting> _watchedSettings = new Dictionary<KeyValueIdentifier, ConfigurationSetting>();
+        private Dictionary<KeyValueIdentifier, IEnumerable<MatchConditions>> _watchedSelectedKeyValueCollections = new Dictionary<KeyValueIdentifier, IEnumerable<MatchConditions>>();
```

Comment:
Names are getting too long. Suggest using `kv` and `ff` for private variables wherever possible.